### DISCUSSION TITLE
Is it possible to have syntax highlighting based on a tree-sitter grammar I provide?

### DISCUSSION THREAD

**Discussion by haikyuu** [zed-industries/zed#6490] [Feature Requests]
I am in the process of writing a tree-sitter grammar. And I want to use it in Zed for syntax highlighting.

Is that currently possible?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
